examples/Example02_CONT_Mode/Example02_CONT_Mode.ino

```diff
@@ -51,14 +53,14 @@ void setup() {
   Serial.println("Sensor began.");
 
   // Set break time to 900us (112 * 8us) to account for the time it takes to poll data.
-  if(SFE_BUS_OK != myUVSensor.setBreakTime(112)) {
+  if(kSTkErrOk != myUVSensor.setBreakTime(112)) {
```

Comment:
I know the break time is explained at the top of the file, but I think it would help to reiterate it here as well for anyone who misses it at the top (like me 😉). Also explain why 900us is chosen

Reply:
...silly pebkacs...

Reply:
But also...

> Also explain why 900us is chosen

`to account for the time it takes to poll data.`

Reply:
I think this is completely unnecessary. I don't need to account for every byte. I measured it and set it to that number experimentally.

Reply:
If you measured it experimentally, then that should be stated in the comment. Eg. "This 900us was experimentally determined to be a suitable amount of time with a 100kHz clock frequency"

> Also where did you get "across 2 separate read transactions"?

My mistake, was thinking it also called `readOutConv()` which reads 4 more bytes, but that's only in `MEAS_MODE_SYND`.

Also, I still got the number of bytes wrong before (oops!), because each read operation actually has to write the register address first. So to read N bytes (6 in this case), a total of N+3 bytes (9 in this case) go across the bus. Working through the timings again, that should theoretically be 810ua at 100kHz, which is in line with your measurement. I still think it's worth offering this explanation in the comment so a user understands _why_ that much time is chosen beyond just experimental findings; it means if they change something (eg. clock frequency or going into `MEAS_MODE_SYND`), then they would know how to change the timing without needing to measure it experimentally.